### ISSUE TITLE
crashes when closing a window

### ISSUE THREAD

**Issue by GreatBrickLab** [rgriebl/brickstore#697]
I have been using the extra view windows for files.  I have 3 splits,  to use 4 in window views and one off window view for the part out of a set.   If you use the red x to close the external windows BS may crash into some kind of a loop,  I think the loop is when the file in the external windows is not saved, and the system popup to ask if you want to save the file tries to open.   
![Screenshot 2023-03-28 090102](https://user-images.githubusercontent.com/76892578/228263925-07fe40b3-d267-4c1a-8f1e-0b35d5fcf76e.png)

**Comment by GreatBrickLab** [comment]
Plus,  if you have the extra window selected you cannot change the view / column layout from there.

**Comment by rgriebl** [comment]
> Plus, if you have the extra window selected you cannot change the view / column layout from there.

Yes - that's a known limitation of the extra windows. As the column layout is per-document, you can just activate the same document in the main-window and change the layout there: it will automatically also change the layout in any other view, including extra windows.

**Comment by rgriebl** [comment]
Are you using the latest BrickStore release? I can't reproduce it, but I know that I had fixed a few problems in that area with the v2023.2.1 release (here: c8de542)
If you still have that problem with v2023.3.2, could you please write down or screen-record the steps to reproduce?

**Comment by GreatBrickLab** [comment]
I am on the latest version and it did occur today.  I tried to recreate it
but failed to do so.  There must be some condition that I am not aware of
yet that is linking the issue.  I will keep you posted for the correct
steps to reproduce the issue.

Chris

**Comment by rgriebl** [comment]
Seems like you don't have Sentry crash reports enabled. Please do enable them in Settings, restart BrickStore and try get it to crash again. This should (hopefully) give me enough information to go on.

**Comment by GreatBrickLab** [comment]
Crash reporting is enabled,  I am closing the program manually.  I will wait to see if the loop will crash out on its own

**Comment by GreatBrickLab** [comment]
> Seems like you don't have Sentry crash reports enabled. Please do enable them in Settings, restart BrickStore and try get it to crash again. This should (hopefully) give me enough information to go on.

It just crashed without me closing it

**Comment by GreatBrickLab** [comment]
the "Open in new window"  crashes in just about every instance now.

**Comment by rgriebl** [comment]
The flickering+crashing on "open in new window" is something different: #712 (introduced in 2023.4.3 and fixed in the current dev build)
Your original problem is different though, as it happens when closing the external window.

**Comment by rgriebl** [comment]
Found this one, going through old issues: was this ever fixed?
